src/components/base/PropertyDrawer.vue

```diff
@@ -1,35 +1,43 @@
 <template>
```

Comment:
One issue I missed out is that, there isn't a handler for different data types! So no checkboxes. I started of with a general model for that and mistakenly though that everything was a slider. It isn't hard to do. Just slipped my mind.